pkg/migration/controllers/migrationschedule.go

```diff
@@ -149,50 +149,61 @@ func (m *MigrationScheduleController) handle(ctx context.Context, migrationSched
 				err.Error())
 			return nil
 		}
-		if migrationSchedule.Spec.AutoSuspend {
-			var remoteMigrSched *stork_api.MigrationSchedule
-			remoteMigrSched, err = remoteOps.GetMigrationSchedule(migrationSchedule.Name, migrationSchedule.Namespace)
-			if errors.IsNotFound(err) {
-				namespace, err := core.Instance().GetNamespace(migrationSchedule.Namespace)
-				if err != nil {
-					return err
-				}
-				namespace.ResourceVersion = ""
-				_, err = coreOps.CreateNamespace(namespace)
-				if err != nil && !errors.IsAlreadyExists(err) {
-					return err
-				}
-				// create new migrationschedule on remote cluster
-				remoteMigrSched = migrationSchedule.DeepCopy()
-				remoteMigrSched.ResourceVersion = ""
-				remoteMigrSched.UID = ""
-				if remoteMigrSched.Annotations == nil {
-					remoteMigrSched.Annotations = make(map[string]string)
-				}
-				remoteMigrSched.Annotations[StorkMigrationScheduleCopied] = "true"
-				suspend := true
-				remoteMigrSched.Spec.Suspend = &suspend
-				remoteMigrSched.Status = stork_api.MigrationScheduleStatus{}
-				if _, err := remoteOps.CreateMigrationSchedule(remoteMigrSched); err != nil {
-					return err
-				}
 
-			} else if err != nil {
+		// create migrationSchedule in remote cluster irrespective of autoSuspend value
+		var remoteMigrSched *stork_api.MigrationSchedule
+		remoteMigrSched, err = remoteOps.GetMigrationSchedule(migrationSchedule.Name, migrationSchedule.Namespace)
+		if errors.IsNotFound(err) {
+			namespace, err := core.Instance().GetNamespace(migrationSchedule.Namespace)
+			if err != nil {
 				return err
 			}
-			if remoteMigrSched.Status.ApplicationActivated {
-				suspend := true
-				migrationSchedule.Spec.Suspend = &suspend
-				msg := "Suspending migration schedule since migrated apps on remote cluster are active"
-				m.recorder.Event(migrationSchedule,
-					v1.EventTypeWarning,
-					"Suspended",
-					msg)
-				log.MigrationScheduleLog(migrationSchedule).Warn(msg)
-				// deactivate apps in namespace
-				// TODO: suspend deploy/sts,crds
-				return m.client.Update(context.TODO(), migrationSchedule)
+			namespace.ResourceVersion = ""
+			_, err = coreOps.CreateNamespace(namespace)
+			if err != nil && !errors.IsAlreadyExists(err) {
+				return err
+			}
+			// create new migrationschedule on remote cluster
+			remoteMigrSched = migrationSchedule.DeepCopy()
+			remoteMigrSched.ResourceVersion = ""
+			remoteMigrSched.UID = ""
+			if remoteMigrSched.Annotations == nil {
+				remoteMigrSched.Annotations = make(map[string]string)
+			}
+			remoteMigrSched.Annotations[StorkMigrationScheduleCopied] = "true"
+			suspend := true
+			remoteMigrSched.Spec.Suspend = &suspend
+			remoteMigrSched.Status = stork_api.MigrationScheduleStatus{}
+			if _, err := remoteOps.CreateMigrationSchedule(remoteMigrSched); err != nil {
```

Comment:
Only return of error is not  AlreadyExists.
Also use the migration schedule copy returned by the server from there onwards instead of `remoteMigrSched`.

Reply:
This particular branch will be entered only if remote migration schedule doesn't already exist. 
So err caught here cannot be AlreadyExists.

2nd point makes sense. Will address.